components/script/dom/headers.rs

```diff
@@ -86,6 +86,7 @@ impl HeadersMethods for Headers {
         if let Some(v) = self.header_list.borrow().get_raw(&valid_name) {
             combined_value = v[0].clone();
             combined_value.push(b","[0]);
+            combined_value.push(b" "[0]);
```

Comment:
I think each element in the raw vector corresponds to a single header value, so instead of doing `v[0].clone()` it seems to me like a new vector should be pushed in `v`, unfortunately it's just a slice and not the underlaying vector. Cc @seanmonstar.